.ci-cd/build.sh

```diff
@@ -70,6 +70,7 @@ function test() {
         alf.algorithms.trac_algorithm_test \
         alf.algorithms.vae_test \
         alf.bin.train_play_test \
+        alf.bin.train_play_test_alfconf.py \
```

Comment:
File name should end with "_test.py" so that `python -m unittest discover -s alf -p "*_test.py" -v` can run all tests. It seems to me most contents of train_play_test and train_play_test_alfconf are same. Perhaps we can just modify train_play_test instead of writing a new one?

Reply:
merged